### PR TITLE
More recent minutes

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

**Comment by gvwilson** [comment]
@karinlag can you please review and merge?

File: minutes/minutes-2015-09-24.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
+# Steering Committee 2015-09-24 DONE
```
Comment (karinlag) [inline minutes/minutes-2015-09-24.md]: Remove DONE

File: minutes/minutes-2015-09-24.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+Motion: That we announce that he's been hired early next week (Greg and Jonah to draft 
+blog post for email approval by Steering Committee).
+
+## 6 Transition Minutes 
+
+Motion: That Jonah Duckles be invited to attend all future meetings of the Steering 
+Committee, subject to a review of the contract by SC and signing by the ED.
+
+(Motion: PASSES)
```
Comment (karinlag) [inline minutes/minutes-2015-09-24.md]: Remove brackets

File: minutes/minutes-2015-09-24.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+
+GitHub have (finally) decided when they’d like the instructor training we owe them - 
+need confirmation that two-day Dec 9-10 is OK.
+
+Motion: Greg (and new instructor trainers if available) do two-day online instructor 
+training for GitHub on Dec 9-10, 2015.
+
+## 7 Instructor Training for GitHub minutes
+
+Both motions passed. (from Karin: was there another motion here too?)
```
Comment (karinlag) [inline minutes/minutes-2015-09-24.md]: Change to:

Motion passed.

File: minutes/minutes-2015-10-08.md [hunk 4]
```diff
@@ -0,0 +1,306 @@
+# Steering Committee 2015-10-08 DONE
```
Comment (karinlag) [inline minutes/minutes-2015-10-08.md]: Remove DONE

File: minutes/minutes-2015-10-08.md [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+between email lists for participants and those for organizers. SCF lists are commonly 
+set up for organizers, not participants. Eventbrite is often used for participants. 
+
+We could go for deleting those that already are inactive. We could also push more towards 
+using Slack, because channels are easy to create and remove. However, any new tool could 
+not work with some university admin. 
+
+Action: leave email list questions to workshop setup proposal
+
+Action: Jonah delete inactive (&gt;3 months or thereabouts) every 6th months.
```
Comment (karinlag) [inline minutes/minutes-2015-10-08.md]: remove/edit greater than

**Comment by karinlag** [comment]
Aplogogies for dropping the ball on this, looks ok barring small changes.

**Comment by gvwilson** [comment]
@karinlag edits done - please merge.

**Pull request merged**